docs/source/quick-start.rst

```diff
@@ -347,3 +347,123 @@ with out-of-band message processing:
+    >>> exc_crud.extra_info_error
+    <tarantool.crud.CrudError object at 0x10a276950>
+    >>> exc_crud.extra_info_error.
+    exc_crud.extra_info_error.class_name  exc_crud.extra_info_error.err         exc_crud.extra_info_error.file        exc_crud.extra_info_error.line        exc_crud.extra_info_error.str       
+    >>> exc_crud.extra_info_error.class_name
+    'InsertError'
+    >>> exc_crud.extra_info_error.str
+    'InsertError: Failed to insert: Duplicate key exists in unique index "primary_index" in space "tester" with old tuple - [3500, 300, "Rob"] and new tuple - [3500, 300, "Rob"]'
+
+    # In case of batch operation (*_many), CrudModuleManyError exception contains both result and errors (if there is a problem with at least one row).
+    >>> try:
+    ...     res = conn.crud_insert_object_many('tester', ({'id':3,'bucket_id':100,'name':'Ann'}, {'id':4,'bucket_id':100,'name':'Sam'}), {'timeout':100, 'rollback_on_error':False})
+    ... except CrudModuleManyError as e:
+    ...     exc_crud = e
+    ... 
+    >>> exc_crud
+    CrudModuleManyError(0, 'Got multiple errors, see errors_list')
+    >>> exc_crud.success_list # some of the rows were inserted.
+    <tarantool.crud.CrudResult object at 0x11a56f310>
+    >>> exc_crud.success_list.rows
+    [[1, 100, 'Bob'], [2, 100, 'Rob']]
+    >>> exc_crud.errors_list # some of the rows were not inserted.
+    [<tarantool.crud.CrudError object at 0x11a56e9e0>, <tarantool.crud.CrudError object at 0x11a56f490>]
+    >>> exc_crud.errors_list[0].str
+    'CallError: Failed for 037adb3a-b9e3-4f78-a6d1-9f0cdb6cbefc: Function returned an error: Duplicate key exists in unique index "primary_index" in space "tester" with old tuple - [3500, 300, "Rob"] and new tuple - [3500, 100, "Mike"]'
+    >>> exc_crud.errors_list[1].str
+    'InsertManyError: Failed to flatten object: FlattenError: Object is specified in bad format: FlattenError: Unknown field "second_name" is specified'
+    
+    # If there are no problems with any rows, the entire response will be contained in the res variable.
+    >>> res = conn.crud_insert_object_many('tester', ({'id':3,'bucket_id':100,'name':'Ann'}, {'id':4,'bucket_id':100,'name':'Sam'}), {'timeout':100, 'rollback_on_error':False})
+    >>> res.rows
+    [[3, 100, 'Ann'], [4, 100, 'Sam']]
+
+If module crud not found on the router or user has not sufficient grants:
+
+.. code-block:: python
+
+    >>> try:
+    ...     res = conn.crud_insert('tester', (22221,300,'Rob'))
+    ... except DatabaseError as e:
+    ...     exc_db = e
+    ... 
+    >>> exc_db
+    DatabaseError(33, "Ensure that you're calling crud.router and user has sufficient grants")
+    >>> exc_db.extra_info
+    BoxError(type='ClientError', file='/tmp/tarantool-20221003-6335-edruh3/tarantool-2.10.3/src/box/lua/call.c', line=112, message="Procedure 'crud.insert' is not defined", errno=0, errcode=33, fields=None, prev=None)
```

Comment:
Don't forget to update here after message rework.

Reply:
Updated